### DISCUSSION TITLE
osu lazer crash on start up

### DISCUSSION THREAD

**Discussion by creative215** [ppy/osu#21869] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

osu basically crash on start up(a lot of times)
when i open osu but if i click on another app ex. google on my 2nd monitor osu will still be running fine BUT if i clicked on osu which is on my first monitor the app will and no audio for a few seconds and then crash with no window not responding thing.
but if i dont click on google, so if i just start osu up it will go to a black screen i can still hear the sounds but no visual output just a pure black screen. so i cant play and i just close by ending task in task manager BUT that is when i realised
![image](https://user-images.githubusercontent.com/78542086/209752597-ef5f0b71-7a61-4c5a-b955-b2016fd9cdab.png)
when osu crashes it eats up my gpu like hell so i think this might help.

and if im lucky i can start osu and play it if its the first time i open it on a pc session

also here are my computer hardware stuff in case you will need it
RTX 3060
ram 32gb
Intel i9-10900f

### Screenshots or videos

the file size is too big so i just zip it
[2022-12-28 10-37-02.zip](https://github.com/ppy/osu/files/10311963/2022-12-28.10-37-02.zip)

video explanation 
first launch i click on obs as you can see and its fine untill i click on osu and it crashed
second launch i click on the app while in start up and it just goes black screen with sound

### Version

every version that i played on

### Logs

[input.log](https://github.com/ppy/osu/files/10311976/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10311977/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/10311978/network.log)
[performance.log](https://github.com/ppy/osu/files/10311979/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10311980/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10311981/updater.log)
[database.log](https://github.com/ppy/osu/files/10311982/database.log)

**Comment by bdach** [comment, marked as answer]
Please update your graphics card drivers and try again. Also please try disabling shadowplay if you have that on.

**Reply by creative215** [reply]
thanks! it is now fixed!
